### ISSUE TITLE
[Quest][StormPeaks][Sons of Hodir] Raising Hodir's Spear

### ISSUE THREAD

**Issue by TheRealestMcCoy** [dalaranwow/dalaran-wow#3988]
**Current behaviour**: Tell us what happens in detail (Where, When, What, Why, Who)
You are to collect everfrost shards in the cavern. However there are no shards. 
**Expected behaviour**: Tell us what should happen instead
Go to cavern collect shards.
**Steps to reproduce the problem**:

1. Be on quest raising hodir's spear
2. Go in cave look for shards.
3. 

**Include proofs for this behaviour**

You can research this on several pages like http://wowhead.com or http://wowwiki.wikia.com or http://www.youtube.com/
Issues without proofs will be closed and tag On Hold will be added until the proof is provided.

**Include the ID for the game objects, npcs (creatures,pets,minions), spells, items, quests, instances, zones, achievements, skills**

You can research this on several pages like http://wotlk.openwow.com or http://wowhead.com/
[Video Proof](https://www.youtube.com/watch?v=31N4IMAkLlE)
[Quest](http://www.wowhead.com/quest=13001/raising-hodirs-spear)
 [Shards](http://www.wowhead.com/item=42541/everfrost-shard
)
**Include Screenshots from the issue if necessary**
Screen Shot of me at collection place in game with no shards.
![wowscrnshot_121816_024824](https://cloud.githubusercontent.com/assets/12252441/21292268/d551e330-c4cc-11e6-8bf5-4ee23dabfd3b.jpg)

**Comment by Rushor** [comment]
DB/Gameobject: Spawn several Everfrost Shards

* closes https://github.com/dalaranwow/dalaran-wow/issues/3988
* commit: b6c6a9e1cb64480e027156d70188cb99dd357b5b

The issue was fixed by a developer and will now be tested on the Public Test Realm (PTR). The changes/fixes will be applied with the NEXT changelog on the liverealm.
This means that the issue exists in game at the moment i wrote this message.
Only once the issue was marked with "Fixed - On Live Server", you can find the fix on you server.
Please watch http://www.dalaran-wow.com/news/ to see when the changelog and the fix was applied.

**Comment by Sweettouch** [comment]
this is still not fixed i'm standing where the shards should be and it's just blank i have stood around it 4 times now to see if it will fix or respawn and there is nothing :/

**Comment by TheRealestMcCoy** [comment]
@Rushor Rushor added the Fixed - Waiting for PTR Update label 18 days ago

**Comment by Sweettouch** [comment]
so what we have to wait for the update or something?

**Comment by TheRealestMcCoy** [comment]
Yes its pending update on the PTR. The it has to be tested their then it will be updated to the main realm.

**Comment by TeridanDW** [comment]
Works as intended on PTR.
Note: The model of the gameobject is not right, I remember added in correct ones in 1 report but I can't find it now :(
